reg/drone/service/common/Arming.0.1.uavcan

```diff
@@ -0,0 +1,52 @@
+# The arming state is used to command or report the availability (readiness) status of a networked service (subsystem).
```

Comment:
I would rename this to `Arm`.

Reply:
Why?

Reply:
`Arming` just seems like a command exectuion. While `Arm` can be a status or a command.

Reply:
"Arming" here is to be understood as "ArmingState". "Arm" is either an extremity of the human body or an imperative.